### PR TITLE
Ignore the retired branch when requesting commit or watch* ACLs

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/pkgdb2/issues/209

### CODE REVIEW

File: pkgdb2/templates/package.html [hunk 1]
```diff
@@ -68,7 +68,6 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-            {{ package.monitoring_status }}
```
Comment (puiterwijk) [inline pkgdb2/templates/package.html]: This doesn't seem related to this change.
Was this a mistake, or another change you just wanted to sneak in at the same time?
(which is fine. Just want to be sure it's intended)

Comment (pypingou) [inline pkgdb2/templates/package.html]: It's intended, didn't think it was worth a separate PR for just this

Comment (puiterwijk) [inline pkgdb2/templates/package.html]: Sure, fair enough.

**Comment by puiterwijk** [comment]
Looks good to me. :+1:

**Comment by puiterwijk** [comment]
Still :+1:

**Comment by pypingou** [comment]
Thanks! :)
